### PR TITLE
feat(web): hide subtitle if does not exist

### PR DESCRIPTION
**Pull request by issmail-basel**
# Overview
When there is no key, do not show anything instead of `undefined`

### CODE REVIEW

File: web/src/components/molecules/Content/List/index.tsx [hunk 1]
```diff
@@ -37,7 +37,7 @@ const ContentListMolecule: React.FC<Props> = ({
     <ContentWrapper modelsMenu={ModelsMenu}>
       <PageHeader
         title={model?.name}
-        subTitle={`#${model?.key}`}
+        subTitle={model?.key ? "#" + model.key : null}
```
Comment (KaWaite) [inline web/src/components/molecules/Content/List/index.tsx:40]:
```suggestion
        subTitle={model?.key ? `#${model.key}` : null}
```

File: web/src/components/molecules/Schema/index.tsx [hunk 2]
```diff
@@ -42,7 +42,7 @@ const Schema: React.FC<Props> = ({
       }
       center={
         <Content>
-          <PageHeader title={model?.name} subTitle={`#${model?.key}`} />
+          <PageHeader title={model?.name} subTitle={model?.key ? "#" + model.key : null} />
```
Comment (KaWaite) [inline web/src/components/molecules/Schema/index.tsx:45]:
```suggestion
          <PageHeader title={model?.name} subTitle={model?.key ? `#${model.key}` : null} />
```

File: web/src/components/molecules/Content/List/index.tsx (hunk 1 above)
Comment (issmail-basel) [reply web/src/components/molecules/Content/List/index.tsx]: @KaWaite , updated.

File: web/src/components/molecules/Schema/index.tsx (hunk 2 above)
Comment (issmail-basel) [reply web/src/components/molecules/Schema/index.tsx]: @KaWaite , updated.

**Review by KaWaite — APPROVED** [review]

**Pull request merged**